app/models/host/base.rb

```diff
@@ -371,6 +371,25 @@ def uptime_seconds
 
     private
 
+    def parse_ip_address(address, ignore_link_local = true)
```

Comment:
nit: `I'd make this a kwarg`.

Reply:
That's like this, right?
```suggestion
    def parse_ip_address(address, ignore_link_local: true)
```

Reply:
Yep.

Reply:
Updated